ci/cases/pr/ATM_cold.yaml

```diff
@@ -0,0 +1,11 @@
+untar:
+    tarball : "{{ ATARDIR }}/{{ cycle_YMDH }}/atm_cold.tar"
```

Comment:
You may want to set up another directory on HPSS.  `ATARDIR` is where archived tarballs will be sent.  You could add a new directory to `config.base` and call it something different (`FETCHDIR`? or whatever fits your fancy).

Reply:
Adding export for FETCHDIR in both versions of config.base